src/web/collectors/CollectorsPage.jsx

```diff
@@ -1,17 +1,14 @@
-import React from 'react';
-import { Link } from 'react-router';
-import { LinkContainer } from 'react-router-bootstrap';
-
 import DocsHelper from 'util/DocsHelper';
 
+import React from 'react';
```

Comment:
I would put this import on the top because it is an external module and not an internal module